$NetBSD: patch-ab,v 1.3 2003/09/11 12:40:24 jmmv Exp $

gtksourceview-0.6.0 (the one in pkgsrc) is not compatible with this version
of libgnomedb.

--- configure.orig	2003-09-11 01:02:05.000000000 +0200
+++ configure
@@ -21026,6 +21026,7 @@ fi
 
 
 
+if false; then
   succeeded=no
 
   if test -z "$PKG_CONFIG"; then
@@ -21120,6 +21121,8 @@ echo "${ECHO_T}$GTKSOURCEVIEW_LIBS" >&6
   else
      have_gtksourceview=no
   fi
+fi
+have_gtksourceview=no
 
 
 
